processor/tailsamplingprocessor/internal/sampling/span_count_sampler_test.go

```diff
@@ -41,26 +42,47 @@ func TestEvaluate_NumberSpans(t *testing.T) {
 			NotSampled,
 		},
 		{
-			"Less spans than the threshold",
+			"Bigger amount of spans than the threshold, in one single batch",
 			[]int32{
-				1, 1, 1,
+				3,
 			},
-			NotSampled,
+			Sampled,
 		},
 		{
-			"Same number of spans than the threshold",
+			"Bigger amount of spans than the threshold, with different span count per batch",
 			[]int32{
 				1, 2, 1,
 			},
 			Sampled,
 		},
 		{
-			"Bigger amount of spans than the threashold",
+			"Same number of spans as the threshold, in one single batch",
+			[]int32{
+				2,
+			},
+			Sampled,
+		},
+		{
+			"Bigger amount of spans than the threshold, across multiple batches",
+			[]int32{
+				1, 1, 1,
+			},
+			Sampled,
```

Comment:
Previously, this was not sampled. This was the bug, right?

Reply:
Yes, that's right